peef/peef.go

```diff
@@ -17,6 +18,8 @@ type Price struct {
 	Volume int32   `json:"volume"`
 }
 
+var Log = logrus.New()
```

Comment:
How I understand it, you export variables in go using Capitalized names. Ideally this would only be exported into the *scope* of the project and not anymore external.